.github/workflows/ci.yml

```diff
@@ -1674,12 +1674,13 @@ jobs:
       env:
         PGP_SECRET: ${{ secrets.PGP_SECRET }}
```

Comment:
I get the feeling that running `gpg-setup.sh` is no longer necessary with this approach... 🤔 Not entirely sure, however. Likely no harm to leave it for now?